src/mutation/RelayMutationQueue.js

```diff
@@ -133,6 +133,18 @@ class RelayMutationQueue {
 
   rollback(id: ClientMutationID): void {
     const transaction = this._get(id);
+    const collisionKey = transaction.getCollisionKey();
+    const collisionQueue =
+        collisionKey && this._collisionQueueMap[collisionKey];
+    if (collisionQueue) {
```

Comment:
Per @josephsavona, doing the check here makes it so you don't have to do it on line 144. Better yet, you can avoid the redundancy by doing:

```
if (collisionKey) {
  const collisionQueue = this._collisionQueueMap[collisionKey];
  if (collisionQueue) {
    // Flow now knows that both `collisionKey` and `collisionQueue` are truthy.
  }
}
```